addons/website_slides/static/src/js/slides_course_fullscreen_player.js

```diff
@@ -660,30 +660,37 @@
         _renderSlide: function () {
             var slide = this.get('slide');
             var $content = this.$('.o_wslides_fs_content');
-            $content.empty();
+
+            /**
+             * Set the widget as the only child of $content.
```

Comment:
Will something like that work ?
```js
if (this._toggleSlideCompletedCalled) return;
this._toggleSlideCompletedCalled = true;
...
this._toggleSlideCompletedCalled = false;
```

It can be done in addition to the current fix, to not trigger useless render. If we call many time the render method it shouldn't add many quizz so we can keep the current fix (we might want to update the commit message if we do that change, because the first click will win, not the last one)

Reply:
Hello @std-odoo , 

Thanks for the review.

Yes that’s a good idea. I have done it on the _renderSlide method because the method _toggleSlideCompleted takes arguments so it would be a bit odds not to execute its body when some part is pending. I have removed the orginal fix then because it becomes unnecessary and updated the commit message to explain the choice.

The only drawback is that I need to do it in a try catch which is changing more lines due to the indentation (and causing security warning on the runbot).